boards/px4/fmu-v3/init/rc.board_sensors

```diff
@@ -15,6 +15,12 @@ hmc5883 -C -T -I -R 4 start
 # Internal SPI bus ICM-20608-G
 mpu6000 -T 20608 start
 
+# External SPI
+ms5611 -S start
+
+# Internal SPI (auto detect ms5611 or ms5607)
+ms5611 -T 0 -s start
```

Comment:
px4/fmu-v3 and px4/fmu-v2 should be kept in sync.

Reply:
Done